src/pages/api-reference/layers/vector-layer.md

```diff
@@ -27,6 +27,17 @@ Extends [`L.Layer`](http://leafletjs.com/reference-{{siteData.latest_leaflet}}.h
 </table>
 
 
+##### Esri Vector Basemaps Group
+
+Esri has developed many vector basemaps.  A complete listing of these is available on [ArcGIS Online](https://www.arcgis.com/home/group.html?id=30de8da907d240a0bccd5ad3ff25ef4a&view=list&start=1&focus=layers#content). Many of these basemaps provide support for local languages at large scales.  These have "Local Language" included in the item name. 
+
+The basemaps provide a mix of content, and can be grouped accordingly:
+- Map features and labels
+- Only map features
+- Only labels
+
+It may be possible to mix standard [basemaps](https://esri.github.io/esri-leaflet/api-reference/layers/basemap-layer.html) with label-only vector basemaps.
+
 ##### Custom Vector Tile Services
 
 Whether you've made a small tweak to one of Esri's hosted vector basemaps or published your own vector tile service from scratch you can draw the layer in Esri Leaflet by referencing the item's [`id`](http://www.arcgis.com/home/item.html?id=bd505ce3efff479bb4e87b182f180159).
```

Comment:
it'd be useful to add a link to the vector tile style editor here.

https://www.esri.com/arcgis-blog/products/developers/mapping/design-custom-basemaps-with-the-new-arcgis-vector-tile-style-editor/